graphx/src/main/scala/org/apache/spark/graphx/impl/GraphImpl.scala

```diff
@@ -265,14 +264,6 @@ class GraphImpl[VD: ClassTag, ED: ClassTag] protected (
     }
   }
 
-  /** Test whether the closure accesses the attribute with name `attrName`. */
-  private def accessesVertexAttr(closure: AnyRef, attrName: String): Boolean = {
```

Comment:
[Handle ClassNotFoundException from ByteCodeUtils](https://github.com/apache/spark/commit/bee1015620de28806660f20d7c9f404dd3bdb2cb) introduce this method and it used by `mapReduceTriplets`, [Code cleaning to improve readability.](https://github.com/apache/spark/commit/d58bfa85738adb57a5311f52f52ede5903a181e1) clean up `mapReduceTriplets` from `GraphImpl.scala`, `mapReduceTriplets` is no longer used